packaging/windows/sharedframework/sharedframework.wxs

```diff
@@ -11,6 +11,27 @@
         <Feature Id="MainFeature" Title="Main Feature" Level="1">
             <ComponentGroupRef Id="InstallFiles" />
             <ComponentGroupRef Id="AuthoredRegistryKeys"/>
+            <Component Id="BreadcrumbStoreFolder" Directory="BreadcrumbStore" Guid="{DA957490-EA59-42B9-B9D6-3E80F6FB0ED0}">
```

Comment:
Keypath is a folder. Tons of Components can root to the same folder and get colliding keys very easily. For example, you could choose to split these permissions into separate Components and easily get collisions.

Stable and unique. Unique is the hard part.